src/Check/ScrutinizerCheck.php

```diff
@@ -7,7 +7,9 @@
 
 class ScrutinizerCheck extends Check
 {
+    // phpcs:disable
     const THRESHOLD = 6.5;
+    // phpcs:enable
```

Comment:
```suggestion
    const THRESHOLD = 6.5;
```
The phpcs issue was resolved in another PR which has since been merged.

Reply:
I am a bit confused about the suggestion and master branch... however, i merged the master into this PR